<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Properties</class>
 <widget class="QDialog" name="Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIPS Properties</string>
  </property>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>385</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>516</width>
     <height>146</height>
    </rect>
   </property>
   <property name="title">
    <string>Startup options</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>115</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MIPS TCP/IP list </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMinMIPS">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>85</y>
      <width>36</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLoadControlPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load control panel</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leControlPanel">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>60</y>
      <width>346</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>216</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum number of MIPS systems</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboTCPIPlist">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>110</y>
      <width>176</width>
      <height>26</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkAutoConnect">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto connect at startup</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbClear">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>107</y>
      <width>116</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear list</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSearchAMPS">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>30</y>
      <width>226</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Search for AMPS, AMS baud rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAMPSbaud">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>30</y>
      <width>86</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>38400</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="chkAutoFileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>155</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatic unique file name selection</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>516</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Directories</string>
   </property>
   <widget class="QPushButton" name="pbDataFilePath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>116</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Data file path</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScriptPath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>75</y>
      <width>116</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Script path</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDataFilePath">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>20</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leScriptPath">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>80</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMethodesPath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>110</y>
      <width>116</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Methodes path</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMethodesPath">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>115</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFileName">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>50</y>
      <width>126</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Test.0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>57</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Base name</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbLogFile">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>340</y>
     <width>116</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Log file</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLogFile">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>345</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seconds between updates</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUpdateSecs">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>390</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkAutoRestore">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>155</y>
     <width>236</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatically restore connections</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
